packages/@aws-cdk/aws-apigateway/lib/domain-name.ts

```diff
@@ -162,4 +178,26 @@ export interface DomainNameAttributes {
    * Thje Route53 hosted zone ID to use in order to connect a record set to this domain through an alias.
    */
   readonly domainNameAliasHostedZoneId: string;
+
+}
+
+/**
+ * The mTLS authentication configuration for a custom domain name.
+ */
+export interface MTLSAttributes {
```

Comment:
Sorry, I missed this in the first round of reviews. We usually use the `Attributes` suffix in conjunction with import APIs - https://github.com/aws/aws-cdk/blob/master/DESIGN_GUIDELINES.md#from-attributes.

What do you think about switching this to `MTLSConfig` instead?